public/app/features/dashboard-scene/settings/AnnotationsEditView.tsx

```diff
@@ -1,29 +1,89 @@
 import React from 'react';
 
-import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { AnnotationQuery, DataTopic, PageLayoutType } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+import { SceneComponentProps, SceneDataLayerProvider, SceneObjectBase, sceneGraph } from '@grafana/scenes';
 import { Page } from 'app/core/components/Page/Page';
 
+import { DashboardScene } from '../scene/DashboardScene';
 import { NavToolbarActions } from '../scene/NavToolbarActions';
+import { dataLayersToAnnotations } from '../serialization/dataLayersToAnnotations';
 import { getDashboardSceneFor } from '../utils/utils';
 
+import { AnnotationSettingsList } from './annotations';
 import { DashboardEditView, DashboardEditViewState, useDashboardEditPageNav } from './utils';
 
-export interface AnnotationsEditViewState extends DashboardEditViewState {}
+export interface AnnotationsEditViewState extends DashboardEditViewState {
+  editIndex?: number | null;
+}
 
 export class AnnotationsEditView extends SceneObjectBase<AnnotationsEditViewState> implements DashboardEditView {
   public getUrlKey(): string {
     return 'annotations';
   }
 
+  constructor(state: AnnotationsEditViewState) {
+    super({
+      ...state,
+      editIndex: null,
+    });
+
+    this.addActivationHandler(() => {
+      this.setState({ editIndex: this.getEditIndex() });
+    });
+  }
```

Comment:
I dont think you need any of this if you implement URL sync similar to i.e. [Dashboard links edit view](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/settings/DashboardLinksEditView.tsx#L23) or [Variables edit view](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/settings/VariablesEditView.tsx#L36) 

```
protected _urlSync = new EditListViewSceneUrlSync(this);
```